spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "POST #create" do
+    it "successfully creates a Post" do
+      post :create, params: { post: { title: "A title", text: "Some text" } }
```

Comment:
Should I also be testing the route somewhere? As in should I be trying to make a POST to `/posts` at some point in these tests?

Reply:
I think this is sufficient 👍